src/components/search/SearchResult.jsx

```diff
@@ -13,6 +13,7 @@ export const SearchResult = ({
   issueAge,
   numComments,
   labels,
+  darkMode
```

Comment:
Just realized I think we don't need to use this prop in this file. I wonder if you remove all the new props passing (so the only change is in the scss file)  if it will still work?

Reply:
Good point! We don't need the props to update the styling: 
![image](https://user-images.githubusercontent.com/40472917/135766966-b6fd3808-d54d-484a-97a4-3525378f575d.png)